packages/block-library/src/post-comments-form/form.js

```diff
@@ -34,7 +34,6 @@ const CommentsForm = () => {
 						name="submit"
 						type="submit"
 						className={ classnames(
-							'submit',
```

Comment:
Where else is this class? If it's not needed should we remove the directives as well?

Reply:
I haven't found the class anywhere else in Gutenberg codebase.
The CSS directives live in the WordPress core repo: https://github.com/WordPress/wordpress-develop/blob/0a17a80bccd452b91c3b63f71f010ba131a2c954/src/wp-admin/css/forms.css#L415-L421